src/services/offer.js

```diff
@@ -0,0 +1,65 @@
+const Offer = require("../models/Offer");
+
+class OfferService {
+    // TODO: Use typedi or similar
+    constructor() {
+
+    }
+
+    async create({
+        title,
+        publishDate,
+        endDate,
+        jobMinDuration,
+        jobMaxDuration,
+        jobStartDate,
+        description,
+        contacts,
+        isPaid,
+        vacancies,
+        jobType,
+        fields,
+        technologies,
+        isHidden,
+        owner,
+        location,
+        coordinates,
+    }) {
+        const offer = await Offer.create({
+            title,
+            publishDate,
+            endDate,
+            jobMinDuration,
+            jobMaxDuration,
+            jobStartDate,
+            description,
+            contacts,
+            isPaid,
+            vacancies,
+            jobType,
+            fields,
+            technologies,
+            isHidden,
+            owner,
+            location,
+            coordinates,
+        });
+
+        return offer;
+    }
+
+    async get() {
```

Comment:
Since the idea was to get this working as fast as possible I did not worry with this for now. Do you feel that this is strictly necessary and should be included here? I would rather include it later, even if in a very small PR, since this PR is already becoming very big.

Reply:
It should be here, it's part of the Offer querying.. and it would not make the PR that much bigger (maybe 5 more lines?)

Reply:
It implies handling params and their validation. Furthermore, I have to look up mongoose docs and I'm lazy 😛

Reply:
That should be solved with 2.3 cups of coffee (on average) :coffee:

Reply:
Done! Please have a look